playbooks/provisioning/openstack/upscaling_scale-up.yaml

```diff
@@ -0,0 +1,27 @@
+---
+# Run provision.yaml with higher number of nodes to create a new app-node VM
+- include: provision.yaml
+
+# Run config.yml to update openshift installation (add the new node(s))
+# TODO kpilatov: let the user specify the path using a variable
+- include: ../../../../openshift-ansible/playbooks/byo/config.yml
+
+# Post-verification: Verify new number of nodes
+# TODO: remove verification based on oc?, keep openstack_num_nodes
+- hosts: masters
+  tasks:
+  - name: Get number of nodes
+    shell: oc get nodes | grep app-node -c
+    register: oc_new_result
+  - name: Check that the actual number of nodes matches defined value
+    assert:
+      that: 'nodes | int == oc_new_result.stdout | int'
+      msg: >
+        FAIL: Number of application nodes has not been increased accordingly
+        (it should be {{ nodes }} but it is {{ oc_new_result.stdout | int }}).
```

Comment:
`{{ nodes }}` isn't set if you just want to increment the scale count by one (and therefore don't explicitly pass `-e nodes=...`.